src/components/survey/generic/container.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from "prop-types";
+import style from "./style.scss";
+
+export default function Container({children, progress}) {
+  return (
+    <div className={`${style.container}`}>
+      {progress < 100
```

Comment:
I'm surprised the linter didn't catch this, but better to have this and the next line on one line:

```
{progress < 100 && (
```